managers/battle_manager.py

```diff
@@ -129,6 +128,7 @@ def _player_death(self, player, monster):
 
     def _handle_game_state_change(self, previous_state, new_state, data):
         if(data["new_state"] == State.STATE_BATTLE):
+            print(data)
```

Comment:
remember to remove this in the future. should have had a "DEBUG" in it so when you search youll find turds like this :\